src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountSelectCardPage.tsx

```diff
@@ -30,6 +30,7 @@ function QuickbooksCompanyCardExpenseAccountSelectCardPage({policy}: WithPolicyC
     const policyID = policy?.id ?? '';
     const {exportCompanyCard, syncLocations} = policy?.connections?.quickbooksOnline?.config ?? {};
     const isLocationEnabled = Boolean(syncLocations && syncLocations !== CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE);
+    const {creditCards, bankAccounts, accountPayable} = policy?.connections?.quickbooksOnline?.data ?? {};
```

Comment:
memo: checked that these field names are correct

<img width="309" alt="Screenshot 2024-05-06 at 9 01 06 AM" src="https://github.com/Expensify/App/assets/98560306/7d802554-ab76-452f-a59e-e2e51e80700c">